doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst

```diff
@@ -43,83 +43,33 @@ On your computer, one of the following operating systems:
-      :alt: Cellular Monitor - Program device
-
-      Cellular Monitor - Program device
-
-   The **Program sample app** window appears, displaying applications you can program to the DK.
-
-#. Click :guilabel:`Select` in the **Asset Tracker V2** section.
-   Asset Tracker v2 is an application that simulates sensor data and transmits it to Nordic Semiconductor's cloud solution, `nRF Cloud`_.
-
-   .. figure:: images/cellularmonitor_selectassettracker.png
-      :alt: Cellular Monitor - Select Asset Tracker V2
-
-      Cellular Monitor - Select Asset Tracker V2
-
-   The **Program Asset Tracker V2** window appears.
-
-#. Click :guilabel:`Program` to program the DK.
-   Do not unplug or turn off the device during this process.
-
-   .. figure:: images/cellularmonitor_programassettracker_nrf9160.png
-      :alt: Cellular Monitor - Program Asset Tracker V2
-
-      Cellular Monitor - Program Asset Tracker V2
-
-   When the process is complete, you see a success message.
-   Click :guilabel:`Close` to close the **Program Asset Tracker V2** window.
+Getting started using Quick Start
+*********************************
 
-#. Copy the :term:`Integrated Circuit Card Identifier (ICCID)` of the inserted micro-SIM.
-   This is required for activating the iBasis SIM when :ref:`nrf9160_gs_connect_to_cloud`.
+To work with the nRF9160 DK firmware and certificates, complete the following steps:
 
-   If you have activated your iBasis SIM card before or are using a SIM card from a different provider, you can skip this step.
+1. Install `nRF Connect for Desktop`_.
+#. Start the nRF Connect for Desktop and install the Quick Start application, a cross-platform tool for guided setup and installation procedures.
+#. Prepare your DK:
 
-   a. Click :guilabel:`Start` to begin the modem trace.
-      The button changes to :guilabel:`Stop` and is greyed out.
-   #. Click :guilabel:`Refresh dashboard` to refresh the information.
-   #. Copy the ICCID by clicking on the **ICCID** label or the displayed ICCID number in the **Sim** section.
+   a. Punch out the nano-SIM from the SIM card and plug it into the SIM card holder on the nRF9160 DK.
+   #. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF91**.
+      On DK v0.9.0 and earlier, this is the **SW5** switch.
+   #. Connect the nRF9160 DK to the computer with a micro-USB cable, and then turn the DK on.
 
-      .. figure:: images/cellularmonitor_iccid.png
-         :alt: Cellular Monitor - ICCID
+#. Use the Quick Start app to update the DK firmware.
+   The recommended actions are the following:
 
-         Cellular Monitor - ICCID
+   * In the Program step, select :guilabel:`Asset Tracking`.
+   * In the Verify step, copy the International Mobile Equipment Identity (IMEI) number.
+     This is needed for connecting to nRF Cloud.
+   * In the SIM Setup step, read and copy the :term:`Integrated Circuit Card Identifier (ICCID)` of the inserted micro-SIM.
```

Comment:
Have you checked if this copies the ICCID with 18 or 20 characters?

Note that this is also referenced on Line 208 (old) / 158 (new).

Reply:
There is now a note in the application that points out that the first 18 digits of ICCID are needed.

Reply:
FYI.
We want to implement the "ICCID automatic read out" directly in the quick start tool in the future so the user do not need to read up on the doc. -> start up another tool -> read out the iccid -> remove 2 digits manually